src/modules/Login/action.js

```diff
@@ -289,6 +290,18 @@ export const mergeSettings = (
     }
   }
 
+  // Filter conflicting tokens out of synced settings:
+  if (finalSettings.customTokens && account != null) {
```

Comment:
Even if we change the denomination info for a currency here, back in the calling `initializeAccount` function one line 183 it can undo the work we've done in this `mergeSettings` routine

Reply:
If you follow the data-flow, `initializeAccount` line 183 is using the data that comes out of `mergeSettings`. Indeed, `mergeSettings` is the very first thing that happens to the settings after they come off of disk, and is therefore the most definitive place to do an edit like this.